### PR TITLE
[DATA-683] Change default webcam selector to mediadevices

### PR DESCRIPTION
**Pull request by kharijarrett**

### CODE REVIEW

**Comment by kharijarrett** [comment]
This PR is a quickie to make sure we're putting our best foot forward when it comes to guessing in the absence of a "video_path." I actually tested this on a Pi with some webcams (bcuz how else?) and this solution already fixes the use case where you have 2 webcams plugged in and no "video_path." Before you would get an empty stream that'll lead to big red errors, now you get whichever webcam claimed video0.  Good start, but we're not done with usability here just yet.

File: components/camera/videosource/webcam.go
Comment (edaniels) [inline components/camera/videosource/webcam.go:212]: this deletes path pattern which we still need

Comment (kharijarrett) [reply components/camera/videosource/webcam.go:212]: Unclear that we do... according to Fahmina, according to the consulting team, zero people are actually using it (see comments: https://docs.google.com/document/d/18p3LV6ghH-m0xGjO7MZRz4zK0X8JePmGZRAXBtm0k7E/edit) and there's not much reason to compare a regex pattern anymore

Comment (bhaney) [reply components/camera/videosource/webcam.go:212]: Once the scope doc is approved, it will remove PathPattern, and this will be OK.

**Review by bhaney — APPROVED** [review]
Looks good, but maybe wait until the scope doc is approved to click "merge"

File: components/camera/videosource/webcam.go
Comment (edaniels) [reply components/camera/videosource/webcam.go:212]: I didn't see this scope

**Review by edaniels — APPROVED** [review]
LGTM pending scope

**Pull request closed without merging**
